### PR TITLE
Add 1.6 Release Notes

### PR DESCRIPTION
**Pull request by kalafut**

### CODE REVIEW

File: website/pages/docs/release-notes/1.6.0.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+state of the Vault servers and saves them locally, or pushes them to an optional remote storage service.
+
+**UI improvements**
+* **New UI for Transform secrets engine (Enterprise ADP Module Only)**
+  * To configure FPE and Masking transformations, including custom alphabets and patterns for FPE.
+  * To create and manage roles and patterns for templated use in current and future FPE and masking transformations.
+* **Improvements to Vault Usage UI** to show new metrics for “active clients”, “unique entities” and “active direct
+tokens” that help with understanding Vault usage
+
+**New Command “vault usage”**: Support for analyzing active usage of applications and users with new metric client
```
Comment (jweissig) [inline website/pages/docs/release-notes/1.6.0.mdx:32]: This should likely be called `New “Vault Usage”` and maybe mention it is UI based. There is not CLI option as this time (only API + UI).

**Comment by jweissig** [comment]
Added a comment, other than that, LGTM.

File: website/pages/docs/release-notes/1.6.0.mdx (hunk 1 above)
Comment (kalafut) [reply website/pages/docs/release-notes/1.6.0.mdx:32]: Ok I’ll remove it.

File: website/pages/docs/release-notes/1.6.0.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+**Transform: Tokenization Tech Preview (Enterprise ADP Module Only)**: Vault 1.6 introduces a new transformation method
+for tokenizing sensitive data stored in un-trusted/semi-trusted systems. Tokenization is available as part of the
+“Advanced Data Protection” module in Vault Enterprise.  Tokenization provides non-reversible data protection pursuant to
+requirements for data irreversibility (PCI-DSS, GDPR, etc.). This feature is being released in a Tech Preview.
+
+**Integrated Storage Enhancements**:  Continuing with the enhancements made to Vault’s Integrated storage, we are adding
+the following new features:
+* **Cloud auto-join** provides support for Vault nodes to automatically discover and join a cluster via specified cloud
+metadata. This is particularly useful when IP addresses are not static.
+* **Automated snapshots** provides in-built Vault functionality for automated snapshots that takes snapshots of the
```
Comment (mladlow) [inline website/pages/docs/release-notes/1.6.0.mdx:22]:
```suggestion
* **Automated snapshots** provides built-in Vault functionality for automated snapshots that takes snapshots of the
```
I haven't heard "in-built" often but don't have strong feelings.

File: website/pages/docs/release-notes/1.6.0.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+state of the Vault servers and saves them locally, or pushes them to an optional remote storage service.
+
+**UI improvements**
+* **New UI for Transform secrets engine (Enterprise ADP Module Only)**
+  * To configure FPE and Masking transformations, including custom alphabets and patterns for FPE.
+  * To create and manage roles and patterns for templated use in current and future FPE and masking transformations.
+* **Improvements to Vault Usage UI** to show new metrics for “active clients”, “unique entities” and “active direct
+tokens” that help with understanding Vault usage
+
+**Support for seal migration** for all use cases to migrate from any to any unseal interface (including auto-unseal to auto-unseal)
```
Comment (mladlow) [inline website/pages/docs/release-notes/1.6.0.mdx:32]:
```suggestion
**Support for seal migration** for all use cases to migrate from any to any unseal interface (including auto-unseal to auto-unseal of the same type)
```
This was specifically what we added in 1.6

**Review by mladlow — APPROVED** [review]
I have a couple suggestions but seems fine. I'll try to catch up with @jweissig around 10AM EST/7AM PST to see if I can merge this before doing the docs cut over

**Pull request merged**